src/main/java/uk/gov/justice/digital/hmpps/whereabouts/dto/VideoLinkBookingResponse.kt

```diff
@@ -19,12 +19,21 @@ data class VideoLinkBookingResponse(
   val agencyId: String,
 
   @ApiModelProperty(
-    value = "The location of the court that requires the appointment",
+    value = "The name of the court that requires the appointment. May be a place-holder",
     example = "York Crown Court",
-    required = true
+    required = true,
+    notes = "At least one of court or courtId will be returned"
   )
   val court: String,
```

Comment:
Using "court" as a court name on input (as per VLBSpecification) and either name or id on return is confusing. But you probably have to do this for backwards compatibility?

Reply:
Yes